mxnet/inference/docker/1.8/py3/sdk1.18.0/Dockerfile.neuron

```diff
@@ -0,0 +1,147 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+#SDK 1.17.1 has version 1
+LABEL dlc_major_version="2"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.1.8
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.10
```

Comment:
Since just updating the neuron sdk version don't want to update the python version. In the new PT 1.11 container release will try this new python version and fix all the issues that are found with using that python version in the test/install failures.